sdk/storage/azure-storage-blobs/samples/blob_list_operation.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "get_env.hpp"
+#include <get_env.hpp>
```

Comment:
Should we specify a more exact path in these, given `get_env.hpp` lives in the `samples` root folder and not under `sdk/storage`?

Reply:
That's the core difference between `"foo"` and `<foo>`.

`<foo>` says "Search in the `INCLUDE` environment variable and in the directories referenced by the `-I` switches. 
`"foo"` says "Search in the current directory, in the directories of any currently open include files, the `INCLUDE` environment variable and any `-I` directories".

Most customers treat `<>` paths to be the paths they get from an SDK, and the `""` paths to be stuff they included in their application.

For samples, if the header comes from a vcpkg, IMHO it should be `<>`.